task/one-shot/Cargo.toml

```diff
@@ -12,6 +12,9 @@ cortex-m-semihosting = { version = "0.5.0", optional = true }
 [target.'cfg(target_arch = "riscv32")'.dependencies]
```

Comment:
Can we use cfg any to provide both riscv32 and riscv64 conditionals here?

Reply:
We can indeed, see latest change.